examples/index.html

```diff
@@ -5,7 +5,6 @@
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
     <meta name="viewport" content="initial-scale=1.0, user-scalable=no, width=device-width">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" type="text/css">
-    <link rel="stylesheet" href="../css/ol.css" type="text/css">
```

Comment:
It was there because when run in the template the examples were not importing ol.css (which was causing inconsistencies in some examples with custom css when viewed on the example page and when run stand-alone or in codesandbox).  Has that changed?  If not it should be included, but for better consistency with the stand-alone behaviour it might be best placed below the `{{{ css.tag }}}`

Reply:
The index.html file is this: https://openlayers.org/en/main/examples/, as there is no map there should be no need for the ol.css.